backend/src/main/kotlin/com/ksidelta/libruch/modules/organisation/UserOrganisationsReadModel.kt

```diff
@@ -11,16 +13,21 @@ import javax.persistence.EmbeddedId
 import javax.persistence.Entity
 
 @Service
-class UserToOrganisationsEventProcessor(val repository: UserToOrganisationsModelRepository) {
+class UserToOrganisationsEventProcessor(
+    val repository: UserToOrganisationsModelRepository,
+    val eventGateway: EventGateway
+) {
 
-    @EventHandler
+    @EventHandler(payloadType = MemberAdded::class)
     fun handle(event: DomainEventMessage<MemberAdded>) {
         repository.save(
             UserToOrganisationsModel(
                 UserAndOrganisation(event.payload.user.id, UUID.fromString(event.aggregateIdentifier)),
                 event.payload.organisationName
             )
         )
+
+        eventGateway.publish(UserToOrganisationsModelViewUpdated())
```

Comment:
At the end of processing query the event is sent.
What is interesting is that UnitOfWork will stop any event from being sent if the transaction fails <3.